Doc/library/functions.rst

```diff
@@ -554,8 +554,9 @@ are always available.  They are listed here in alphabetical order.
    If the given source is a string, then leading and trailing spaces and tabs
    are stripped.
 
-   See :func:`ast.literal_eval` for a function that can safely evaluate strings
-   with expressions containing only literals.
+   .. versionchanged:: 3.11
```

Comment:
I'm not so sure about saying 'version changed', as it seems to imply that only 3.11 has the change, but 3.10 also has it as it was backported ([see PR](https://github.com/python/cpython/pull/95919)). I'd say it's okay to just keep the original format and adjust the wording.

Reply:
i'll do that